knative/Tiltfile

```diff
@@ -0,0 +1,91 @@
+# -*- mode: Python -*-
+
+# Configuration for all knative CRDs.
+k8s_kind('Service', api_version='serving.knative.dev/v1',
+         image_json_path='{.spec.template.spec.containers[*].image}')
+
+def knative_install():
+  """
+  Installs the knative operators into your cluster.
+
+  Auto-configures knative to use any local dev registries.
+  """
+
+  k8s_custom_deploy(
+    'knative-crds',
+    deps=[],
+    apply_cmd="""
+set -e
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.0.0/serving-crds.yaml -o yaml
+kubectl wait --for=condition=Established crd -l=knative.dev/crd-install=true 1>&2
+""",
+    delete_cmd="""
```

Comment:
I could see it being useful to have an arg for `knative_install` to control down policy for for the CRDs/knative core, but not critical by any means

Reply:
yahhhh... i could see a couple different 'down' approaches:
- delete the whole knative namespace
- delete nothing at all!

but also thinks this dovetails a lot with the work we're doing around resource subsets